back/lib/shared/hello/hello_test.go

```diff
@@ -0,0 +1,25 @@
+package hello_test
+
+import (
+	. "airneis/lib/shared/hello"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+)
+
+func TestHello(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecsWithDefaultAndCustomReporters(t, "Hello", []Reporter{reporters.NewJUnitReporter("test_report_hello.xml")})
+}
+
+var _ = Describe("HelloTest", func() {
+	Context("When we want to print hello", func() {
+		word, number := HelloWorld()
+		It("should return a string with hello world and 25 as an integer", func() {
+			Expect(word).To(Equal("HelloWorld"))
+			Expect(number).To(Equal(13))
+		})
+	})
+})
```

Comment:
Could you remove this file in another commit as it's content is on the readme
But thx for providing test sample

Reply:
Sure I can!

Reply:
I think to I'll keep it for now to not remove the packages when making a "go mod tidy", I'll remove it in the next PR when I add other real tests cases.